docs/users-manual/submitting-a-job.rst

```diff
@@ -1182,6 +1190,13 @@ as configured by the administrator of the Vault server:
 
     use_oauth_services = dune_production
 
+.. warning::
```

Comment:
There's probably something to be said about Vault roles here, but I'm still not clear yet on how roles come in to play, so probably better to omit rather than lie 🙂 .